depthai_ros_driver/src/param_handlers/stereo_param_handler.cpp

```diff
@@ -56,28 +56,38 @@ void StereoParamHandler::declareParams(std::shared_ptr<dai::node::StereoDepth> s
     stereo->setLeftRightCheck(declareAndLogParam<bool>("i_lr_check", true));
     int width = 1280;
     int height = 720;
-    dai::CameraBoardSocket socket = dai::CameraBoardSocket::RIGHT;
     if(declareAndLogParam<bool>("i_align_depth", true)) {
-        try {
-            width = getROSNode()->get_parameter("rgb.i_width").as_int();
-            height = getROSNode()->get_parameter("rgb.i_height").as_int();
-            socket = static_cast<dai::CameraBoardSocket>(getROSNode()->get_parameter("rgb.i_board_socket_id").as_int());
-        } catch(rclcpp::exceptions::ParameterNotDeclaredException& e) {
-            RCLCPP_ERROR(getROSNode()->get_logger(), "RGB parameters not set, defaulting to 1280x720 unless specified otherwise.");
-        }
-
-    } else {
-        try {
-            width = getROSNode()->get_parameter(rightName + ".i_width").as_int();
-            height = getROSNode()->get_parameter(rightName + ".i_height").as_int();
-            socket = static_cast<dai::CameraBoardSocket>(getROSNode()->get_parameter(rightName + ".i_board_socket_id").as_int());
-        } catch(rclcpp::exceptions::ParameterNotDeclaredException& e) {
-            RCLCPP_ERROR(getROSNode()->get_logger(), "Right parameters not set, defaulting to 1280x720 unless specified otherwise.");
+        switch(static_cast<dai::CameraBoardSocket>(declareAndLogParam<int>("i_board_socket_id", static_cast<int>(dai::CameraBoardSocket::RGB)))) {
+            case dai::CameraBoardSocket::RGB:
+                stereo->setDepthAlign(dai::CameraBoardSocket::RGB);
+                try {
+                    width = getROSNode()->get_parameter("rgb.i_width").as_int();
+                    height = getROSNode()->get_parameter("rgb.i_height").as_int();
+                } catch(rclcpp::exceptions::ParameterNotDeclaredException& e) {
+                    RCLCPP_ERROR(getROSNode()->get_logger(), "RGB parameters not set, defaulting to 1280x720 unless specified otherwise.");
+                }
+                break;
+            case dai::CameraBoardSocket::LEFT:
+                stereo->setDepthAlign(dai::StereoDepthProperties::DepthAlign::RECTIFIED_LEFT);
```

Comment:
when the case is CameraBoardSocket::LEFT. it should be a real camera that is on the board. 
RECTIFIED_LEFT refers to a virtual camera made from the LEFT camera. 
So to align to that you need to use [this](https://docs.luxonis.com/projects/api/en/latest/references/python/?highlight=RECTIFIED_LEFT#depthai.RawStereoDepthConfig.AlgorithmControl.DepthAlign.RECTIFIED_LEFT)